src/components/Products/GridItem/GridItem.js

```diff
@@ -15,11 +21,15 @@ const GridItem = ({ imageUrl, name, price, id }) => {
   };
 
   const showProductHandler = () => {
-    console.log(id);
+    navigate(`/product/${id}`);
   };
 
   const showDetailModalHandler = () => {
-    console.log('show detail modal');
+    setIsDetailModalVisible(true);
+  };
```

Comment:
These two methods indicate that something will be displayed, but `showProductHandler` is redirecting to another page

Reply:
I changed first name to: __redirectToDetailPage__